build/generate-protobuf-files.sh

```diff
@@ -20,6 +20,8 @@ mkdir -p ./generated-js
 rm -rf ./generated-js/*
 mkdir -p ./generated-java
 rm -rf ./generated-java/*
+mkdir -p ./generated-python
```

Comment:
Please [add this to `.gitignore`](https://github.com/netspeak/netspeak4-application-ngrams/blob/b322c19e22c71f39be95b37b14f43daab1cb9024/.gitignore#L13).